css/signin.css

```diff
@@ -7,47 +7,77 @@ box-sizing: border-box;
 }
 
 body {
-margin: 50px auto;
-background-color: rgba(241, 241, 241, 0.9);
+background-color: #fff;
 font-family: 'Poppins', sans-serif;
 font-weight: 700;
+overflow: hidden;
 }
 
-h1 {
-text-align: center;
-color: #1F59BB;
-padding-top: 20px;
+.container {
+display: flex;
+height: 900px;
+
 }
 
-.wrapper {
-width: 550px;
+.info-wrap {
+background: #1F59BB;
+width: 50%;
 display: block;
-background-color: #fff;
-position: absolute;
-left: 35%;
-top: 17%;
-margin-left: -55px;
-box-sizing: border-box;
-border-radius: 5px;
-margin-bottom: 60px;
+}
+
+.logo {
+    padding: 25px 60px;
+    margin-top: 7px;
+    height: 100px;
+}
+
+h1 {
+    color: #fff;
+    padding-left: 60px;
+    font-size: 40px;
+}
+
+h5 {
+    color: #fff;
+    font-weight: 300;
+    padding: 10px 60px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5689283)